.github/workflows/test-core.yaml

```diff
@@ -185,12 +185,20 @@ jobs:
       - name: Benchmark Secure Cell (master key)
         run: |
           cd benches/themis
-          cargo bench -- 'Secure Cell .* master key/4 KB'
+          cargo bench -- '^Secure Cell .* master key/4 KB'
       - name: Benchmark Secure Cell (passphrase)
         run: |
           cd benches/themis
           # These are awfully slow due to KDF
-          cargo bench -- 'Secure Cell .* passphrase/4 KB' --sample-size 10
+          cargo bench -- '^Secure Cell .* passphrase/4 KB' --sample-size 10
+      - name: Benchmark Secure Message (RSA)
+        run: |
+          cd benches/themis
+          cargo bench -- 'Secure Message .* - RSA/4 KB'
```

Comment:
Should these maybe start with `^Secure Message` too, just in case RustThemis is going to get Secure Message benchmarks?

Reply:
Yeah, makes sense!